
  Don't draw scroll thumb if the terminal is large enough to not
  need one. Reported by Martin von Gagern <Martin.vGagern@gmx.net>.

--- ufed-0.40/ufed-curses.c
+++ ufed-0.40/ufed-curses.c
@@ -109,7 +109,8 @@
 	wattrset(w, COLOR_PAIR(3) | A_BOLD);
 	mvwaddch(w, 0, 0, ACS_UARROW);
 	wvline(w, ACS_CKBOARD, wHeight(Scrollbar)-3);
+	if(items->prev->top+items->prev->height > wHeight(List))
-	mvwaddch(w, 1+(wHeight(Scrollbar)-3)*topy/(items->prev->top+items->prev->height-(wHeight(List)-1)), 0, ACS_BLOCK);
+		mvwaddch(w, 1+(wHeight(Scrollbar)-3)*topy/(items->prev->top+items->prev->height-(wHeight(List)-1)), 0, ACS_BLOCK);
 	mvwaddch(w, wHeight(Scrollbar)-2, 0, ACS_DARROW);
 	mvwaddch(w, wHeight(Scrollbar)-1, 0, ACS_VLINE);
 	wnoutrefresh(w);
@@ -380,7 +381,9 @@
 			mousekey = c; \
 		goto check_key; \
 	}
+						if(items->prev->top+items->prev->height > wHeight(List))
+							{}
-						if(event.y == 0)
+						else if(event.y == 0)
 							SIM(UP)
 						else if(event.y == wHeight(Scrollbar)-2)
 							SIM(DOWN)
